ipatests/test_ipalib/test_util.py

```diff
@@ -0,0 +1,44 @@
+# Authors:
+#   Oleg Kozlov <okozlov@redhat.com>
+#
+# Copyright (C) 2018  Red Hat
+# see file 'COPYING' for use and warranty information
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Test the `ipalib.util` module.
```

Comment:
You also copied another bad example. In Python doc strings the short description / title is on the first line after the triple quotes. There is no additional space between header and module doc string, too.

```
#
# Copyright (C) 2015  FreeIPA Contributors see COPYING for license
#
"""Tests for ipalib.util module
"""
```